.gitlab-ci.yml

```diff
@@ -5,14 +5,16 @@ stages:
 
 variables:
     WS_ARTIFACT_PATH: eva-accession-ws/target/eva-accession-ws-$ENVIRONMENT_NAME.war
-    CLUSTERING_ARTIFACT_PATH: eva-accession-clustering/target/eva-accession-clustering-*.jar
+    CLUSTERING_ARTIFACT_PATH: eva-accession-clustering/target/eva-accession-clustering-*exec.jar
```

Comment:
Why are some of our jars annotated with `exec` ?

Reply:
When a module (in this case, the eva-accession-clustering module) serves as both a dependency for another module (in this case, the eva-accession-deprecate module) and also an executable (for the clustering pipeline), it has to have two deployed JARs: one that can make it behave as a "library" to be used as a dependency and another (with the exec suffix) that can be an "executable" (the one that will be deployed).